fuzz/client.c

```diff
@@ -50,8 +50,6 @@ int FuzzerInitialize(int *argc, char ***argv)
     ERR_get_state();
     CRYPTO_free_ex_index(0, -1);
     idx = SSL_get_ex_data_X509_STORE_CTX_idx();
-    RAND_add("", 1, ENTROPY_NEEDED);
-    RAND_status();
```

Comment:
I want the global state before and after FuzzerTestOneInput() to be exactly the same. That means if someone internally calls RAND_bytes() or RAND_status(), it should not alter any global variable.

If you say that it's "like the rest of OpenSSL", it probably means it will be initialized on first use, which is fine. The code there is so that this first time has happened, and that this initialization doesn't happen during the first (or 100th) call to FuzzerTestOneInput(), and then next call this whole thing doesn't get executed anymore. The point of calling those functions is that the global state doesn't change anymore. Maybe calling RAND_add() isn't the best way to accomplish this (anymore), maybe it should now just have a call to RAND_bytes(). But in any case it's doubtful that just removing those lines will have the desired effect.